src/PlatformNames.jl

```diff
@@ -155,19 +155,19 @@ function platform_key(machine::AbstractString = Sys.MACHINE)
     if startswith(machine, "x86_64-apple-darwin")
         return MacOS()
     end
-    if ismatch(r"x86_64-(pc-)?(unknown-)?linux-gnu", machine)
+    if ismatch(r"x86_64-(pc-)?(unknown-)?(redhat-)?linux(-gnu)?", machine)
```

Comment:
Is `x86_64-unknown-redhat-linux` valid? If not, the `unknown` might need to be `(unknown-|redhat-)?`.

Reply:
Not sure. Someone else would have to chime in.